src/components/child-pages.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import navItems from '@primer/gatsby-theme-doctocat/src/nav.yml'
+import {Box, Heading, Link, Text} from '@primer/react'
+import {graphql, Link as GatsbyLink, useStaticQuery} from 'gatsby'
+
+type NavItems = Array<
+  | NavItem
+  | {
+      title: string
+      children: Array<
+        | NavItem
+        // Max depth is 2
+        | {
+            title: string
+            children: Array<NavItem>
+          }
+      >
+    }
+>
+
+type NavItem = {
+  title: string
+  url: string
+}
+
+export function ChildPages({of: title}: {of: string}) {
+  const childItems = getChildItems(navItems, title)
+  const data = useStaticQuery(graphql`
+    query MyQuery {
+      allSitePage {
+        nodes {
+          path
+          context {
+            frontmatter {
+              description
+            }
+          }
+        }
+      }
+    }
+  `)
+
+  const descriptionsByPath = data.allSitePage.nodes.reduce((acc, node) => {
+    acc[node.path] = node.context.frontmatter?.description || ''
+    return acc
+  }, {})
+
+  return (
+    <Box sx={{display: 'grid', gap: 5}}>
+      {childItems.map(item => (
+        <Box key={item.url} sx={{display: 'grid', gap: 1}}>
+          <Heading as="h3" sx={{fontSize: 2}}>
+            <Link as={GatsbyLink} to={item.url}>
+              {item.title}
+            </Link>
+          </Heading>
+          <Text as="p" sx={{m: 0, maxWidth: '60ch'}}>
```

Comment:
can we increase the maxWidth here? Feels a little narrow on wide screens
before:
<img width="1518" alt="Screenshot 2023-03-21 at 7 44 09 PM" src="https://user-images.githubusercontent.com/586552/226766554-a4373c90-ea86-40c6-9647-4d82af2a5d55.png">


after (80):
<img width="1268" alt="Screenshot 2023-03-21 at 7 43 58 PM" src="https://user-images.githubusercontent.com/586552/226766567-b852bd8a-b868-4c4d-944c-a5d6f424573e.png">


Separately - we should center this content but this is outside of this PR